zookeeper/include/zk_nn_client.h

```diff
@@ -112,41 +123,67 @@ class ZkNnClient : public ZkClientCommon {
    * @param zk_in shared pointer to a ZKWrapper
    * @return ZkNnClient
    */
-  explicit ZkNnClient(std::shared_ptr<ZKWrapper> zk_in);
+  explicit ZkNnClient(std::shared_ptr<ZKWrapper> zk_in,
```

Comment:
can you add documentation for the changed method signatures in this file

Reply:
@yukichul